### PR TITLE
Followup for the NCR loadouts

### PR DESCRIPTION
**Pull request by Pebblesthe2nd**
## About The Pull Request

Some stuff that did not make it in time for the big one, wich was also big enough as it was.
Logistics (LO-CE-Log RE ) getting final touches for the current style, with CE being the salvage focus combat heavy version with fun gimmick guns/bombs, Rear ech  being the poorly equipped semi-civvie tinker OR salvager, and the LO being master of both salvage and gun tinker.

This leaves the NCR with only 1-5 roles with master gunsmith, wich is more than plenty.
BP wise, salvaged PA and Entrenching tool moved to the three Logistics/engineering roles instead of for all, technical stuff SHOULD be their domain, thats their whole. reason for existing. 

Rear ech getting some love, with a nice utility vest thats worse armor but got two pockets. 
More interesting and makes more sense than them having the frontline infantry armor, looks better  too.
Off-duty gets to glide around in a dress jacket with his saved up money and a nine mil but little else, it just looks nice and makes it different from the other loadouts in a fun way.

Cap loses his smoke grenades, both frags and smokes is a bit much and it makes him mirror the Legion setup too much.

While mucking about with pockets, noticed the bootcode, and adjusted so the fullsize police revolver and needler dont fit in a boot now more, but the gunknife does instead. The detective .38 still fits, despite it being silly, because its a kind of fun gimmick and gives it a unique niche.

Adjusts the hobo guns malfunction rate back to 1% for them all.

### CODE REVIEW

File: code/datums/components/storage/concrete/pockets.dm [hunk 1]
```diff
@@ -56,8 +56,7 @@
 		/obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
 		/obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
 		/obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
-		/obj/item/firing_pin, /*/obj/item/throwing_star/throwingknife, obj/item/melee/cultblade/dagger,*/ /obj/item/gun/ballistic/revolver/detective,
-		/obj/item/gun/ballistic/revolver/police, /obj/item/gun/ballistic/revolver/needler
+		/obj/item/firing_pin, /obj/item/gun/ballistic/revolver/detective, /obj/item/gun/ballistic/revolver/hobo/knifegun
```
Comment (Rohesie) [inline code/datums/components/storage/concrete/pockets.dm:59]:
```suggestion
		/obj/item/firing_pin,
		/obj/item/gun/ballistic/revolver/detective,
		/obj/item/gun/ballistic/revolver/hobo/knifegun,
```

File: code/modules/jobs/job_types/ncr.dm [hunk 2]
```diff
@@ -595,6 +592,17 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
 		),
 		)
 
+/datum/outfit/job/ncr/f13combatengineer/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
```
Comment (Rohesie) [inline code/modules/jobs/job_types/ncr.dm:596]:
```suggestion
	. = ..()
```

File: code/modules/jobs/job_types/ncr.dm [hunk 3]
```diff
@@ -428,10 +425,10 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
 
 	loadout_options = list( // ALL: Binoculars, Bowie knife
-	/datum/outfit/loadout/rangerrecon, // DKS Sniper rifle, .45 Revolver
-	/datum/outfit/loadout/rangertrail, // Trail carbine, 2 x .357 Revolvers
-	/datum/outfit/loadout/rangerpatrol, // Scout carbine, .44 SA Revolver
-	/datum/outfit/loadout/rangerpatrolcqb, // Auto-5, .44 Snubnose revolver
+	/datum/outfit/loadout/rangerrecon,		// DKS Sniper rifle, .45 Revolver
+	/datum/outfit/loadout/rangertrail,		// Trail carbine, 2 x .357 Revolvers
+	/datum/outfit/loadout/rangerpatrol,		// Scout carbine, .44 SA Revolver
+	/datum/outfit/loadout/rangerpatrolcqb,	// Auto-5, .44 Snubnose revolver
```
Comment (Rohesie) [inline code/modules/jobs/job_types/ncr.dm:431]: If you could avoid this kind of pattern, it'd be good.

File: code/modules/jobs/job_types/ncr.dm [hunk 4]
```diff
@@ -1050,26 +1054,30 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
+// Corpsman		Chemistry, simple medical
 /datum/outfit/loadout/rearcorps
 	name = "Corpsman"
+	suit = /obj/item/clothing/suit/armor/f13/utilityvest
 	belt = /obj/item/storage/belt/medical
-	gloves = /obj/item/clothing/gloves/color/latex/nitrile
+	gloves = /obj/item/clothing/gloves/f13/leather/fingerless
 	neck = /obj/item/storage/belt/holster/legholster
-	backpack_contents = list(
+	backpack_contents = list(
```
Comment (Rohesie) [inline code/modules/jobs/job_types/ncr.dm:1064]:
```suggestion
	backpack_contents = list(
```

**Comment by VEN0M5** [comment]
Salvaging rear echelon got no salvaging skill or granter

**Comment by KillerGryphon** [comment]
Pebbles, you need to add back the loadouts for the captain. You can keep your additions, but it was a style thing, and you removed their primary gun. The Captain should get a primary, the R91 was balanced for the NCR Leadership role. See this pastebin to make it easier: https://pastebin.com/T9v9hg3K

It makes no sense for the Leadership role to NOT get a primary rifle, IMO, that is my biggest issue with this.

**Comment by Pebblesthe2nd** [comment]
Merged the code edits, and made some final tweaks, listed below.

* Captain gets two loadout, either with a trench knife and El Capitan, a 14mm(.50) Desert Eagle, or with a standard Deagle and the Court Martial, a custom rocket hammer. (credit to Frok for the hammers name) . Both unique weapons got unique stats and icons.
* "light" sergeant swaps gun with the combat medic. Makes no sense the medic ended up with the better gun, as he is not supposed to be a primary fighter like the sgt. No overall change in power.
*MO get his custom m1911 back. It makes little sense that almost none use the standard 9 mil as sidearm, but its just a cool gun and its a shame to remove it. Doesnt really matter balance wise either way, only devalues the MPs gun slightly but he got his traits and baton thats unique.
*Salvager Rear Ech gets the techno skillbook. The rear ech got mechanical differences and purposes thats varied and well defined, don't see a need for any more changes.
*LO gets utility vest instead of the heavier armor. Frontline fighting is the CE:s job, the LO is the best builder/crafter with most explosives and tons of mats, his armor should not be the same as the CE.

**Pull request merged**